apps/cow-fi/hooks/useSetupPage.ts

```diff
@@ -1,6 +1,6 @@
 'use client'
 
-import { useInitializeUtm } from 'modules/utm'
+import { useInitializeUtm } from 'modules/utm/hooks'
```

Comment:
The export remains the same

```suggestion
import { useInitializeUtm } from 'modules/utm'
```

Reply:
Addressed.